index.js

```diff
@@ -42,6 +42,7 @@ class ServerlessPlugin {
 
       var include = false
       var exclude = false
+      var { processEnvOnly: processEnvOnly = false } = this.config
```

Comment:
Minor: this should work too
```suggestion
      var { processEnvOnly = false } = this.config
```